services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, User, Ticket
+
+
+@transaction.atomic
+def create_order(
+        tickets: list,
+        username: str,
+        date: str = None
+) -> None:
+    user = User.objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+    else:
+        order.created_at = "2020-11-10 14:40"
+    order.save()
```

Comment:
Just remove this hardcoded part and all be good😉
```suggestion
    if date:
        order.created_at = date
        order.save()
```